src/views/RecordingsView.vue

```diff
@@ -174,7 +174,8 @@ export default {
       // Build a collection of tagItems - one per animal
       const tagItems = {};
       for (const tag of tags) {
-        const animal = tag.animal === null ? 'F/P' : tag.animal;
+        // z_ to make these types of tags come last on sort
+        const animal = tag.animal === null ? "z_" + tag.event : tag.animal;
```

Comment:
There's already a nice way of sorting tags just below. There's no need to layer another sorting method on top.